src/components/Avatar/Avatar-story.js

```diff
@@ -64,4 +69,59 @@ storiesOf('Components|Avatar', module)
         </div>
       </div>
     )),
+  )
+  .add(
+    'custom default image',
+    withInfo(`
+      This is an example of how to override the default image shown by the avatar element
+    `)(() => (
+      <React.Fragment>
+        <pre>
+          {
+            'NewDefaultIcon = ({ height, width, ...rest }) => <span style={{ fontSize: height/4, width }}>{JSON.stringify(rest)}</span>'
```

Comment:
Could you adapt this comment to work in the `README.md` for Avatar, rather than in the preview frame?